src/ir.cpp

```diff
@@ -7647,7 +7647,7 @@ static TypeTableEntry *get_error_set_intersection(IrAnalyze *ira, TypeTableEntry
         assert(errors[error_entry->value] == nullptr);
         errors[error_entry->value] = error_entry;
     }
-    ZigList<ErrorTableEntry *> intersection_list = {};
+    ZigList<ErrorTableEntry *> intersection_list = {0};
```

Comment:
You will see this in a few places. `ZigList` needs to be initialized in order to guarantee the members are zeroed.

Reply:
Can you cite a source showing that `{0}` is necessary and `{}` is not sufficient?

Reply:
Actually you are right: https://stackoverflow.com/a/1069634/1930331. Maybe I'm thinking about C, but C++ will work.